app/components/UI/TransactionElement/index.js

```diff
@@ -5,6 +5,7 @@ import { fontStyles } from '../../../styles/common';
 import FAIcon from 'react-native-vector-icons/FontAwesome';
 import { strings } from '../../../../locales/i18n';
 import { toDateFormat } from '../../../util/date';
+import TransactionDetails from './TransactionDetails';
```

Comment:
the changes in this file reverting its state to be the same as main right?

Reply:
Yup!